### ISSUE TITLE
 Error when checking input: expected conv2d_21_input to have 4 dimensions, but got array with shape (1, 1, 224, 224, 3)

### ISSUE THREAD

**Issue by harsh2ai** [keras-team/keras#12596]
`import cv2
    from keras.preprocessing import image
    r = cv2.imread('C:\\...\\modi.jpg',1)
    orig = r.copy()
    r = cv2.resize(r, (224, 224))
    r = r.astype("float") / 255.0
    r = image.img_to_array(r)
    ry= np.expand_dims(k, axis=1)`
    from keras.models import load_model
    #model = load_model('atrs')
     (notSanta, santa) = model.predict(r)[0]

i got shape of ry as (1, 1, 224, 224, 3) inspite of adding one dimension it gave me two dimensions in case you wanted to know shape of r it is (400, 650, 3)
